src/views/admin/AllOperations.vue

```diff
@@ -0,0 +1,189 @@
+                    transactionInfo: {
+                        contractName: UC4Identifier.CONTRACT_MATRICULATION,
+                        transactionName: UC4Identifier.TRANSACTION_ADD_MATRICULATION,
+                        parameters: "SS2020 Computer Science v3",
+                    },
+                    state: OperationStatus.PENDING,
+                    reason: "Mocked Reason",
+                    existingApprovals: {
+                        users: [],
+                        groups: [],
+                    } as ApprovalList,
+                    missingApprovals: {
+                        users: [],
+                        groups: ["admin"],
+                    } as ApprovalList,
+                } as Operation,
+                {
+                    operationId: "PendingMatriculation1",
+                    initiator: "MockUser4",
+                    initiatedTimestamp: "2011-10-05T14:48:00.000Z",
+                    lastModifiedTimestamp: "2020-10-05T16:48:00.000Z",
+                    transactionInfo: {
+                        contractName: UC4Identifier.CONTRACT_MATRICULATION,
+                        transactionName: UC4Identifier.TRANSACTION_ADD_MATRICULATION,
+                        parameters: "SS2020 Computer Science v3",
+                    },
+                    state: OperationStatus.PENDING,
+                    reason: "",
+                    existingApprovals: {
+                        users: [],
+                        groups: ["admin"],
+                    } as ApprovalList,
+                    missingApprovals: {
+                        users: [],
+                        groups: [],
+                    } as ApprovalList,
+                } as Operation,
+            ];
+
+            let mockedWatchedOps = [
+                {
+                    operationId: "PendingMatriculation1",
+                    initiator: "MockUser4",
+                    initiatedTimestamp: "2011-10-05T14:48:00.000Z",
+                    lastModifiedTimestamp: "2020-10-05T16:48:00.000Z",
+                    transactionInfo: {
+                        contractName: UC4Identifier.CONTRACT_MATRICULATION,
+                        transactionName: UC4Identifier.TRANSACTION_ADD_MATRICULATION,
+                        parameters: "SS2020 Computer Science v3",
+                    },
+                    state: OperationStatus.PENDING,
+                    reason: "",
+                    existingApprovals: {
+                        users: [],
+                        groups: ["admin"],
+                    } as ApprovalList,
+                    missingApprovals: {
+                        users: [],
+                        groups: [],
+                    } as ApprovalList,
+                } as Operation,
+            ];
+
+            async function requestData() {
+                busy.value = true;
+                await getUserInfo();
+                await getOperations();
```

Comment:
parallelize as shown in another comment

Reply:
Outdated, as the user information is not needed in this component any more